src/app/static-pages/landing-page/sections/about-section/about-section.component.html

```diff
@@ -19,11 +19,7 @@
     <img height="200px" src="assets/images/illustrations/the-team.svg" />
   </div>
 
-  <!-- Add link and remove overlay once trustchecking page is ready -->
-  <div class="page-link-item page-link-item-bittersweet">
-    <div class="page-link-item page-link-item-overlay">
-      <p class="title">Coming soon...</p>
-    </div>
+  <div class="page-link-item page-link-item-bittersweet" [routerLink]="['/trust-checking']">
```

Comment:
Ah, das hatte ich in #158 auch schon rausgenommen

Reply:
aber Du hast dazu noch das .overlay im css entfernt, ich nicht  ...